eodag/plugins/search/base.py

```diff
@@ -91,8 +91,8 @@ def discover_product_types(self) -> Optional[Dict[str, Any]]:
         return None
 
     def discover_queryables(
-        self, product_type: Optional[str] = None
-    ) -> Optional[Dict[str, Tuple[Annotated[Any, FieldInfo], Any]]]:
+        self, product_type: Optional[str] = None, **kwargs: Optional[Dict[str, Any]]
```

Comment:
- the return type is still Optional[Dict[str, Tuple[Annotated[Any, FieldInfo], Any]]]
- **kwargs type hint should be Any
-  docstring missing parameters description